src/libs/services/NetService.spec.ts

```diff
@@ -1,12 +1,10 @@
 import NetInfo, { NetInfoState } from '@react-native-community/netinfo'
 
 import strings from '/constants/strings.json'
-
-import { NetService } from './NetService'
-
+import { NetService } from '/libs/services//NetService'
```

Comment:
Two `/`?

Reply:
good catch, weird that I didn't have a failing test, or the CI

Reply:
updated